_runners/vault.py

```diff
@@ -72,14 +83,18 @@ def _get_policies(minion_id, config):
   #mappings = { 'minion': minion_id, 'grains': __grains__, 'pillar': minion_pillar }
   mappings = { 'minion': minion_id, 'grains': grains or {}}
 
+  my_policies = _get_my_policies()
```

Comment:
I agree that it add a new request each time a token is requested, but I don't see where and how cache the result. There is no current state nor initialisation in the runner.